src/CommandLineParser.cs

```diff
@@ -13,58 +14,131 @@ public class CommandLineParser
                 return null;
             }
 
-            if (args[0].Contains('-'))
+            CommandLineOptions options = new() { InputPath = "" };
+            for (int i = 0; i < args.Length; i++)
             {
-                if (args.Contains("-v") || args.Contains("--version"))
+                switch (args[i])
                 {
-                    options.ShowVersion = true;
-                    return options;
-                }
+                    case "-v":
+                    case "--version":
+                        options.ShowVersion = true;
+                        return options;
+                    case "-h":
+                    case "--help":
+                        options.ShowHelp = true;
+                        return options;
+                    case "-o":
+                    case "--output":
+                        if (i + 1 < args.Length)
+                        {
+                            // Low priority: If the output path is not specified through the config file, use the one specified through CLI
+                            options.OutputPath ??= args[i + 1];
+                            i++; // Skip the next argument as it is the output file path
+                        }
```

Comment:
if the output path is not specified through the config file, we should use the default `./til` directory. 

generally speaking, if the `-c` flag is used, all other flags in the cli must be ignored.